plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -281,9 +282,10 @@ private JdbcTableHandle flushAttributesAsQuery(ConnectorSession session, JdbcTab
         groupingSetsAsJdbcColumnHandles.stream()
                 .flatMap(List::stream)
                 .distinct()
-                .peek(groupKey -> verify(
-                        tableColumns.map(columns -> columns.contains(groupKey)).orElse(true),
-                        "applyAggregation called with a grouping column which was not included in the table handle"))
+                .peek(handle.getColumns().<Consumer<JdbcColumnHandle>>map(columns ->
+                        groupKey -> verify(columns.contains(groupKey),
```

Comment:
this belongs to `Include grouping sets in JdbcTableHandle column list` commit

Reply:
fmt -- all args on one line, or each on sepearate line (including the first one)